pandas/io/formats/style.py

```diff
@@ -3209,7 +3209,7 @@ def set_properties(self, subset: Subset | None = None, **kwargs) -> Styler:
         return self.applymap(lambda x: values, subset=subset)
 
     @Substitution(subset=subset)
-    def bar(
+    def barchart(
```

Comment:
We can't actually change this since its part of the API - can you exclude this from pylint?

Reply:
Thanks, same with PlotAccessor?